homeassistant/components/zwave.py

```diff
@@ -171,6 +205,15 @@ def value_added(node, value):
             # Ensure component is loaded
             bootstrap.setup_component(hass, component, config)
 
+            # Configure node
+            name = "{}.{}".format(component, _object_id(value))
+
+            customize = config[DOMAIN].get(CONF_CUSTOMIZE, {})
+            node_config = customize.get(name, {})
+            polling_intensity = node_config.get(CONF_POLLING_INTENSITY)
```

Comment:
You could leverage the `util.convert` method here to make sure the returned number is a valid `int`:
```python
polling_intesity = util.convert(node_config.get(CONF_POLLING_INTENSITY), int))
```

If it can't be converted to `int` it will return `None` (or 3rd argument if passed in)